packages/jest-snapshot/src/index.ts

```diff
@@ -310,3 +311,9 @@ export = {
   toThrowErrorMatchingSnapshot,
   utils,
 };
+/* eslint-disable-next-line no-redeclare */
```

Comment:
would be cool to teach this rule about namespaces. Maybe worth an issue to https://github.com/typescript-eslint/typescript-eslint?

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/60